test/fixtures/flow/regression/issue-58-ambiguous/actual.js

```diff
@@ -0,0 +1,4 @@
+// This can be parsed in two ways:
+// a ? b : (c => ((d): e => f))
+// a ? ((b): c => d) : (e => f)
+a ? (b) : c => (d) : e => f;
```

Comment:
Here throwing is the most future-compatible behavior. Probably we should ask to the flow team how to handle this cases.